Dockerfile

```diff
@@ -13,14 +29,19 @@ COPY CMakeLists.txt /root/history-tools
 COPY libraries /root/history-tools/libraries
 COPY src /root/history-tools/src
 COPY unittests /root/history-tools/unittests
+
 RUN mkdir /root/history-tools/build
 WORKDIR /root/history-tools/build
-RUN cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/postgresql ..
-RUN ninja && ctest --output-on-failure
+RUN cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DPostgreSQL_TYPE_INCLUDE_DIR=/usr/include/postgresql .. && \
+    ninja && \
+    ctest --output-on-failure
```

Comment:
I think the test step should be a separate step in Buildkite.

Reply:
It is an option but since this is following the same approach from fill-queue Dockerfile, I think we can leave it as is for now and ask Huang Ming to gather more details.